DOL.WHD.Section14c.Business/Services/EmployerService.cs

```diff
@@ -26,6 +26,22 @@ public EmployerService(IEmployerRepository employerRepository)
             _employerRepository = employerRepository;
         }
 
+        /// <summary>
+        /// Validate employer data
+        /// </summary>
+        /// <param name="employer"></param>
+        /// <returns></returns>
+        public bool ValidateEmployer(Employer employer)
+        {
+            bool validated = true;
+            if (string.IsNullOrEmpty(employer.EIN) || string.IsNullOrEmpty(employer.LegalName) || employer.PhysicalAddress == null ||
+                string.IsNullOrEmpty(employer.PhysicalAddress.StreetAddress) || string.IsNullOrEmpty(employer.PhysicalAddress.City) ||
+                string.IsNullOrEmpty(employer.PhysicalAddress.State) || string.IsNullOrEmpty(employer.PhysicalAddress.ZipCode) ||
+                string.IsNullOrEmpty(employer.PhysicalAddress.County))
```

Comment:
What value is "county" for DC or territories?  Is it set to `N/A`?

Reply:
There will be N/A set for territories and Districts